extensions/panache/rest-data-panache/deployment/src/main/java/io/quarkus/rest/data/panache/deployment/utils/ResourceName.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.rest.data.panache.deployment.utils;
+
+import static io.quarkus.runtime.util.StringUtil.camelHumpsIterator;
+import static io.quarkus.runtime.util.StringUtil.lowerCase;
+import static io.quarkus.runtime.util.StringUtil.toList;
+import static io.quarkus.runtime.util.StringUtil.withoutSuffix;
+
+public final class ResourceName {
+
+    private static final String[] SUFFIXES = { "controller", "resource" };
+
+    public static String fromClass(Class<?> resourceClass) {
```

Comment:
This seems to be unused now, no?

Reply:
True, I had a use for it initially. I'll get rid of it.

Reply:
Removed